.gitignore

```diff
@@ -53,6 +53,7 @@ coverage.xml
 *.py,cover
 .hypothesis/
 .pytest_cache/
+cov.xml
```

Comment:
**thought**: i also noticed this file. I would try to track down what is creating it and stop it.

Reply:
I had assumed it was a coverage report that a CI tool was going to ingest for it's reporting...? I saw it get created, and then I saw the same entry in conductor's .gitignore so I added it here assuming the file itself was useful in CI.

Reply:
ok yes, pytest creates the cov.xml. The coverage.xml file was the duplicate in the other project I was working on.